Aptoide/aptoide-client-v8
# [MOB-71] Migrate event table room

What does this PR do?

This PR aims at implementing the event room table.

Database changed?

No

Where should the reviewer start?

 RoomEvent.java
How should this be manually tested?
Try sending events to aptoide bi. Check in charles if they are correctly sent.

What are the relevant tickets?

Tickets related to this pull-request: MOB-71

Questions:

Does this add new dependencies which need to be added? (Eg. new keys, etc.)

Code Review Checklist

 Documentation on public interfaces
 Database changed? If yes - Migration?
 Remove comments & unused code & forgotten testing Logs
 Codestyle
 New Kotlin code has unit tests
 New flows in presenters unit tests
 Mappers/Validators with any kind of logic unit tests
 Functional tests pass
